samples/hello/scripts/generate-keys.py

```diff
@@ -52,12 +51,29 @@ async def main(config_path, api_key, device_name):
     # Create and self-sign a corresponding public key / certificate
     run(f'openssl req -x509 -new -nodes -key "{root_ca_key}" -sha256 -subj "/C=US/CN=Root {ROOT_CA_NAME}" -days 1024 -out "{root_ca_crt}"')
 
+    if upload:
+        # Add (upload) generated CA certificate to Golioth
+        with root_ca_crt.open('rb') as fp:
+            await project.certificates.add(fp.read(), cert_type='root')
+
+@click.command()
+@click.option('--device-id', help="Primary Device Hardware ID (e.g. a serial number)",
+              required=True)
+@click.option('--project-id', help="Project slug", required=True)
+@click.option('--root-ca-crt', type=Path, help="Root certificate that will sign device certificate",
+              required=True)
+@click.option('--root-ca-key', type=Path, help="Root private key that will sign device certificate",
+              required=True)
+@click.option('-o', '--output', type=Path, default=Path('.'), help="Output directory")
+async def device(device_id, project_id, root_ca_crt, root_ca_key, output):
+    client_name = output / f"{project_id}-{device_id}"
```

Comment:
`client_name` is a path, so this variable name is confusing.

Reply:
Good catch!